src/types.ts

```diff
@@ -36,3 +29,75 @@ export type DraftResponse = number[] | string | null;
 export interface SubmittedResponse {
   answer: number[] | string | null;
 }
+
+export interface QuizMetadata {
+  quiz_type: quizType;
```

Comment:
camelCase?

Reply:
the value here is coming from the backend - toh waha se we are sending in this case only - we have this similar dynamic between fetched values and actual variables to whom those values are assigned in Plio as well.